.changeset/fast-cars-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cube-creator/core-api": patch
+---
+
+Fix removing link between dimension and managed dimension
```

Comment:
I would remove this time. The whole feature has been unreleased so far, so there is technically no bug to fix

Reply:
Yeah... that's why I'm not a fan of semi-automated changelogs ;)
Will remove